pages/api/v1/progressReport/editProgressReportSummary.ts

```
@@ -36,14 +37,14 @@ const handler: NextApiHandler<{ message: string; progress_report?: object }> = a
 		}
 
-		const { postId, proposalType, summary } = req.body;
+		const { postId, proposalType, summary, reportId } = req.body;
 
-		if (!postId || !proposalType || !summary) {
-			return res.status(401).json({ message: messages.INVALID_PARAMS });
+		if (!postId || !proposalType || !reportId) {
+			return res.status(400).json({ message: messages.INVALID_PARAMS });
 		}
 
 		if (isNaN(postId) || !isProposalTypeValid(proposalType)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)